flink-end-to-end-tests/flink-sql-client-test/pom.xml

```diff
@@ -69,6 +69,13 @@ under the License.
 			<artifactId>kafka</artifactId>
 			<scope>test</scope>
 		</dependency>
+
+		<dependency>
+			<groupId>commons-codec</groupId>
+			<artifactId>commons-codec</artifactId>
+			<scope>test</scope>
+		</dependency>
+
```

Comment:
Hi @mbalassi, @slfan1989,

I think we needn't this dependency, the `commons-codec`'s dependency is because `commons-compress` incorrectly depended on `commons-codec`'s Charsets in version 1.26.0. 

This issue has been fixed in version 1.26.1, so perhaps we should bump `commons-compress` version to 1.26.1.
 
see: https://issues.apache.org/jira/browse/COMPRESS-659

Reply:
I have rechecked the dependency of `commons-codec` in `commons-compress` and it is no longer optional. Even if upgraded to 1.26.1, `commons-codec` will still be a transitive dependency. 
Please ignore the previous noise, sorry for the disturbance.